tests/14-proxy-ingress.sh

```diff
@@ -16,26 +16,27 @@ CLIENT_LABEL="id.client"
 cleanup
 logs_clear
 
-docker network inspect $TEST_NET 2> /dev/null || {
```

Comment:
aha you are not worried about the patches you mean the actual *.sh files, sure I can do that. Follow up PR though.